drv/gimlet-seq-server/src/main.rs

```diff
@@ -536,6 +524,31 @@ cfg_if::cfg_if! {
         #[cfg(target_board = "gimlet-b")]
         const FPGA_HACK_PINS: Option<&[(sys_api::Port, u16, bool)]> = None;
 
+        //
+        // SP_TO_SP3_UARTA_OE_L must be driven low to allow for transmission
+        // into the SP3's UART
+        //
+        const UART_TX_ENABLE_PIN: (sys_api::Port, u16) =
+            (sys_api::Port::A, 1 << 5);
+
+        fn uart_sp_to_sp3_enable() {
+            let sys = sys_api::Sys::from(SYS.get_task_id());
+            let (port, pin_mask) = UART_TX_ENABLE_PIN;
+
+            sys.gpio_configure(
```

Comment:
You can use `sys.gpio_configure_output` here to simplify

Reply:
Done -- thank you!